docs/ROADMAP.md

```diff
@@ -0,0 +1,84 @@
+
+## Versions
+
+Aligning on language of versions:
+
+* **v0.1**: refers to the initial buildout that will be submitted to the app stores, aiming for Tuesday, March 24
+* **v0.2+**: any versions starting with v0 are _PRE_ publishing the app for public download
+* **v1.0**: the first version that we feel offers enough value to be published onto the app stores for public download will be v1.0
+
+### v0.1
+
+**Goal:** To get a functional build of an application that can be submitted to the Apple App Store and Google Play Store for approval. This app will _NOT_ be published into those app stores for public download, but will instead serve as an approved base for future versions and to prove out that we can deliver on the technical side. The app will serve static content pulled from the WHO's site in English.
+
+**Target Date:** Tuesday, March 24
+
+Technical Features / Requirements:
+
+* App built to be submitted on Apple App Store and Google Play Store
+* Displaying static information in English
+  * Currently, build to design spec in Figma; this may be updated over the course of the day
+* UI styled to exact design spec in Figma
+* App is still functional in an offline environment with default content able to be accessed without a network connection
+* Demonstrate capability to localize content from the technical platform perspective, even if other languages/cultures are _NOT_ part of that v0.1 launch
+  * Framework in place to localize UI elements
+  * Framework in place to load the informational content in localized formats
+* Demonstrate capability to load content dynamically from the cloud, even if this feature will _NOT_ be turned on as part of the v0.1 launch
+* Demonstrate capability to instrument the app and collect basic analytics around which features and pages are actually being used, even if this feature will _NOT_ be turned on as part of the v0.1 launch
+  * Investigating legal requirements of this data collection, including whether updated TOS/Privacy Policy is needed
+
+Features _NOT_ included in v0.1:
+
+* Localization of initial content across languages/cultures outside of English / US
+  * Blockers: translation & adaptation of content to enough other languages/cultures to be a meaningful initial feature
+  * We are building in the capabilities into v0.1 from the technical perspective
+* Triage survey telling users whether or not they should go to the hospital
+  * Blockers: full definition on what the survey entails + legal review on the implications of making medical suggestions
+  * _Question:_ do we want to link to an existing triage survey in a webview for v0.1, and if so, which one?
+* Contact tracing
```

Comment:
Do we even need this here?

Reply:
I think it's important to have given that it's in the original project brief that a lot of people saw when joining the project